### PR TITLE
Clarification on how to provide answers

### PR DESCRIPTION
**Pull request by motomod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,91 @@
...
+Your repository should contain a library for interacting with a REST API. It is highly desired that you provide tests, to prove that your solution works and is robust. 
+
+Alongside your library, you will need some additional logic to take the API responses and convert them into the outputs specified in the scenarios below.
 
 ## Scenario 1
 
 wi-Q is integrating with a fictitious company called 'Great Food Ltd'. 'Great Food Ltd' provide menu and item data from a REST API. Your job is to write some code to consume their API and parse the response into a readable format.
 
-Using the API endpoints described below, write code that would be able to connect to this API and consume the product data for menu with name `Takeaway`. Once the product data has been obtained, print it out in a list, containing the product id and name, such as:
+Using the API endpoints described below, write code that would be able to connect to this API and consume the product data for menu with name `Takeaway`. Once the product data has been obtained, print it out in a list, containing the product id and name.
```
Comment (alister) [inline README.md]:
```suggestion
Using the API endpoints described below, write code that would be able to make a request to this API and collect the product data for the menu named `Takeaway`. Once the product data has been obtained, print it out in a list, containing the product id and name.
```

File: README.md [hunk 2]
```diff
@@ -1,82 +1,91 @@
...
 
+### Expected output
```
Comment (alister) [inline README.md]:
```suggestion
### Expected (sample) output
```

File: README.md [hunk 3]
```diff
@@ -1,82 +1,91 @@
...
 
 | ID | Name    |
 | -- | ------- |
 | 4  | Burger  |
 | 5  | Chips   |
 | 99 | Lasagna |
```
Comment (alister) [inline README.md:27]:
```suggestion
* 4: Burger
* 5: Chips
* 99: Lasagna
```

File: README.md [hunk 4]
```diff
@@ -1,82 +1,91 @@
...
 
+### API Endpoints
+
 The available endpoints for their API are the following:
```
Comment (alister) [inline README.md]:
```suggestion
The available endpoints for the 'Great Food Ltd' API are as follows:
```

File: README.md [hunk 5]
```diff
@@ -1,82 +1,91 @@
...
+> ### /menu/{menu_id}/products
+> #### Request Type
+> `GET`
+> #### Headers
+> Authorization:
+> | Name          | Value          |
+> | ------------- | -------------- |
+> | Authorization | Bearer {token} |
+> #### Response
+> This has been provided in `responses/menu-products.json`
```
Comment (alister) [inline README.md]:
```suggestion
> The list of products for the `Takeaway` menu has been provided in `responses/menu-products.json`
```

File: README.md [hunk 6]
```diff
@@ -1,82 +1,91 @@
...
 
 ## Scenario 2
 
 A customer has been in touch and advised you that product with id 84 in menu 7 has the wrong name. The product is currently named 'Chpis' but it should be named 'Chips'.
 
 Using any of the API endpoints from Scenario 1 and the new method detailed below, write code to demonstrate this item being updated in the 'Great Food Ltd''s API.
```
Comment (alister) [inline README.md]:
```suggestion
Using any of the API endpoints from Scenario 1 and the new method detailed below, write code to demonstrate this item being updated in the 'Great Food Ltd' API.
```

File: README.md [hunk 7]
```diff
@@ -1,82 +1,91 @@
...
- | Name          | Value          |
- | ------------- | -------------- |
- | Authorization | Bearer {token} |
+### Expected output
+
+Proof that the API request has been successful.
+
+> ### /menu/{menu_id}/product/{product_id}
+> #### Arguments
+> Product model as described in `GET` `/menu/{menu_id}/products` response
```
Comment (alister) [inline README.md]:
```suggestion
> Product model as described in the `GET /menu/{menu_id}/products` response
```

**Review by alister — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (motomod) [reply README.md:27]: I don't really see the benefit in this change.

**Review by alister — APPROVED** [review]

**Pull request merged**